### PR TITLE
Handler typo

### PR DESCRIPTION
**Pull request by marcoemorais-aws**
### Motivation
- The `service_name` should appear before `command` in service job handlers, but are transposed.  As a result, the handler will always fail.

#### Change summary
- Reorder so that `service_name` appears before `command` in service job handlers. 

#### Revision diff summary
N/A

### Testing
 **Is your change tested? If not, please justify the reason.**  
 **Please list your testing steps and test results.** 
- CI test run result: <link>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/config/Config.cpp [hunk 1]
```diff
@@ -614,6 +614,12 @@ bool PlainConfig::Jobs::LoadFromJson(const Crt::JsonView &json)
         enabled = json.GetBool(jsonKey);
     }
 
+    jsonKey = JSON_KEY_HANDLER_DIR;
```
Comment (HarshGandhi-AWS) [inline source/config/Config.cpp:617]: why was this added here again? I think you might not be using the latest code.

**Review by HarshGandhi-AWS — APPROVED** [review]

**Review by xlcheng1 — APPROVED** [review]
